src/core/src/op/multiply.cpp

```diff
@@ -51,7 +51,7 @@ bool Multiply::evaluate(TensorVector& outputs, const TensorVector& inputs) const
                                       this,
                                       outputs,
                                       inputs,
-                                      OV_PP_ET_LIST(f32, f64, i32, i64, u32, u64),
+                                      OV_PP_ET_LIST(f32, f64, i8, i32, i64, u8, u32, u64),
```

Comment:
Why do we need these changes?

Reply:
to support `i8` & `u8` reference

Reply:
I meant why do we need to support i8/u8 reference in this PR ;) It seems like we don't need these changes. However, I was just curious, don't see anything bad in that. We can leave it as is